book/src/ui_usage.md

```diff
@@ -87,6 +87,82 @@ Siren provides the ability to exit/withdraw your validators via the validator ma
+
+![deposit-step-6](imgs/ui-dep-6.png)
+
+### Consolidate Validator
+
+`EIP-7251` increases the `MAX_EFFECTIVE_BALANCE` limit up to `2048 ETH`, allowing validators with `0x02` withdrawal credentials to consolidate funds from multiple exited validators. Siren facilitates requests to a consolidation contract, enabling validators to upgrade their withdrawal credentials and merge several validators into one compounding target validator.
+
+![consolidation-target](imgs/consolidation-target.png)
+
+#### Eligibility requirements for consolidation
+
+- Validators must have at least `0x01` withdrawal credentials. Validators with `0x00` credentials must first perform a [BLS Execution Change](./ui_usage.md#validator-bls-withdrawal-credentials).
+
+- Target validators with `0x01` withdrawal credentials must initiate a self-consolidation request to upgrade credentials to `0x02`, enabling them to accept funds and benefit from the increased balance cap.
+
+- Validators selected for merging must already be eligible for withdrawal.
+
+![consolidation-source](imgs/consolidation-source.png)
+
+#### Post-consolidation
+
+- Consolidated validators will exit automatically, and their funds will be transferred to the target validator.
+
+- Validators consolidated under the new credentials (`0x02`) will no longer participate in automatic partial withdrawal sweeps. Instead, withdrawal requests must be explicitly submitted to the withdrawal contract as defined in `EIP-7002`.
+
+### Partial Validator Withdrawal
+
+`EIP-7002` enables partial withdrawals from validators with `0x02` withdrawal credentials and balances exceeding the `MIN_ACTIVATION_BALANCE`. Additionally, validators with upgraded `0x02` credentials will no longer participate in the automatic withdrawal sweeps, making this tool very valuable for Lighthouse validators.
+
+In order to request a partial withdrawal you must have access to the wallet set in the validator's withdrawal credentials and enough ETH to cover the withdrawal request and gas fees. Connect this wallet to the Siren dashboard to start withdrawing funds. All pending withdrawals will be visible in the same view for your convenience.
+
+![partial-withdrawal](imgs/partial-withdrawal-siren.png)
+
+### Partial Validator Top-ups
+
+If your validator's `EFFECTIVE_BALANCE` drops, or you've upgraded to `0x02` compounding withdrawal credentials, you can add additional funds. Simply connect any wallet to Siren and transfer the desired amount directly to your validators.
```

Comment:
We might need to elaborate here. Specify how exactly to transfer money to validators.